compiler/rustc_feature/src/unstable.rs

```diff
@@ -606,6 +608,10 @@ declare_features! (
     (unstable, return_type_notation, "1.70.0", Some(109417)),
     /// Allows `extern "rust-cold"`.
     (unstable, rust_cold_cc, "1.63.0", Some(97544)),
+    /// Allows use of contracts attributes.
+    (unstable, rustc_contracts, "CURRENT_RUSTC_VERSION", Some(133866)),
+    /// Allows access to internal machinery used to implement contracts.
+    (unstable, rustc_contracts_internals, "CURRENT_RUSTC_VERSION", Some(133866)),
```

Comment:
```suggestion
    /// Allows use of contracts attributes.
    (incomplete, contracts, "CURRENT_RUSTC_VERSION", Some(133866)),
    /// Allows access to internal machinery used to implement contracts.
    (incomplete, contracts_internals, "CURRENT_RUSTC_VERSION", Some(133866)),
```

As a lang experiment, these should go in with the experimental warning.  Also, the `rustc_` bit should be removed from the feature flag.

Reply:
Should I go ahead and rename all usages of RustcContract with just Contract?

I also just noticed that `contracts_internals` feature should be in fact marked as `internal`. So I'll do that instead.